cli/Makefile

```diff
@@ -18,10 +18,30 @@ test:
 	$(MAKE) kinda-quick-tests
 	$(PYTEST) -n auto -m 'not quick and not kinda_slow' tests
 
+# Run the end-to-end tests only. These are tests that invoke the semgrep
+# command line, excluding unit tests. They don't include the quality assurance
+# (QA) tests because they take longer and aren't as narrowly targeted.
+#
+# We exclude the few tests that don't invoke the semgrep command directly.
+.PHONY: e2e
+e2e:
+	$(PYTEST) -n auto -v \
+	  -m 'not no_semgrep_cli and not todo' tests/e2e
+
 # Run the extra tests on public repos and semgrep-rules
 .PHONY: qa
 qa:
-	$(PYTEST) -n auto tests/qa
+	$(PYTEST) -n auto -v tests/qa
+
+# Run the end-to-end tests using mlgrep instead of the Python CLI.
```

Comment:
not a big fan of mlgrep. I think osemgrep was better.
or semgrep-ocaml, to differentience from semgrep-python and semgrep-core.

Reply:
I wanted to avoid a long name. I'm fine with `osemgrep`.

Reply:
Renamed.